### PR TITLE
New package: Aakash.Swiss version 1.0.0

### PR DESCRIPTION
**Pull request by aakasheoran**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155220)

### CODE REVIEW

**Comment by aakasheoran** [comment]
> @aakasheoran please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: manifests/a/Aakash/Swiss/1.0.0/Aakash.Swiss.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: Aakash.Swiss
+PackageVersion: 1.0.0
+InstallerType: portable
+Commands:
+- swiss
+Installers:
+- InstallerUrl: https://github.com/aakasheoran/swiss/releases/download/v1.0.0/swiss.exe
+  Architecture: x64
+  InstallerSha256: B085F6AD8380F079C6015C5D353F5DFD72E85601744EC7F88D56B3E6A6F02329
```
Comment (Trenly) [inline manifests/a/Aakash/Swiss/1.0.0/Aakash.Swiss.installer.yaml:12]:
```suggestion
  InstallerSha256: B085F6AD8380F079C6015C5D353F5DFD72E85601744EC7F88D56B3E6A6F02329
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

image

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
This package doesn't respond to most commands, but the [source](https://github.com/aakasheoran/swiss/blob/main/src/main.rs) reveals `-v` gives version data, and `cat` is the main command to interact.

**Review by stephengillie — APPROVED** [review]

**Comment by aakasheoran** [comment]
@stephengillie I will be updating the cli in future versions to provide more descriptive usage info and support for help command.
May you let me know what commands you're referring to here: `This package doesn't respond to most commands`?
It would be helpful for me and the users of the cli tool as well.

**Comment by stephengillie** [comment]
Hi @aakasheoran,

CLI applications frequently have a `/?` or `--help` or similar command, which displays other commands that can be used.

**Comment by aakasheoran** [comment]
Yeah @stephengillie thanks, I will be adding it in future updates.

**Pull request merged**